<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Stride Length</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="strideLengthSlider">
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Gait Phase</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="gaitPhaseSlider">
      <property name="minimum">
       <number>-90</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Duty Factor (Hind)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="dutyFactorHindSlider">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Duty Factor (Fore)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="dutyFactorForeSlider">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Cycles</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="cyclesSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>91</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="strideLengthLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="gaitPhaseLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dutyFactorHindLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dutyFactorForeLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="cyclesLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>strideLengthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>strideLengthLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
